packages/orchestrator-ui-components/src/api/axios.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020-2021 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import axios, { AxiosRequestConfig } from 'axios';
+
+const apiPath = 'https://orchestrator.dev.automation.surf.net/api';
```

Comment:
The api path is also available in the config-context.

Reply:
True. It was more involved to be able to make it accessible in the app but I have introduced a ApiContextProvider and we are reading it from the context now